tests/test_models_user.py

```diff
@@ -0,0 +1,113 @@
+from nose.tools import *
+
+from framework import auth
+from framework.auth import exceptions
+from website import models
+from tests import base
+from tests import factories
+
+
+class UserTestCase(base.OsfTestCase):
```

Comment:
My intention was to move the User model tests there in time, yes. The `test_models.py` is huge and was slowing down my testing cycle, even just to parse it for the tests I wanted.

I like the idea of `test_models` (or, `/tests/models/user.py` if nose would detect the test), but I didn't want to make that big a change without consulting others.

I'll go ahead and move this when I get a chance.